src/main/java/com/minecolonies/colony/ColonyManager.java

```diff
@@ -1,6 +1,9 @@
 package com.minecolonies.colony;
 
 import com.minecolonies.achievements.ModAchievements;
+import com.minecolonies.blocks.AbstractBlockHut;
+import com.minecolonies.blocks.BlockHutBaker;
+import com.minecolonies.blocks.ModBlocks;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.minecolonies.blocks.ModBlocks'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)